doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,186 @@
+# Solve a system of inequalities of a single variable algebraically
+
+Use SymPy to solve a system of univariate inequalities algebraically. For
+example, solving $x^2 < \pi$, $x > 0$ yields $0 < x < \sqrt{pi}$.
+
+Alternatives to consider:
+- For multivariate systems (more than one symbol), try this [SciPy linear
+  programming
+  function](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.linprog.html)
+
+Here is a simple example of solving a system of inequalities of a single
+variable algebraically. solve {func}`~.solve` accepts a list or tuple of
+inequalities to be solved as a system:
+
+```py
+>>> from sympy import symbols, solve, pi
+>>> x = symbols('x')
+>>> solve([x >= 0, x**2 <= pi], x)
+(0 <= x) & (x <= sqrt(pi))
+```
+
+## Guidance
+
+### Include the variable to be solved for in the function call
+
+We recommend you include the variable to be solved for as the second argument
+for {func}`~.solve`. While {func}`~.solve` can currently solve systems of
+equations with only one symbol, it is a good practice in case that capability is
+expanded, and because {func}`~.solve` can solve an equation with more than one
+symbol.
+
+## Solve a system of inequalities algebraically
+
+You can solve a system of inequalities algebraically in several ways.
+
+### Enter your inequalities directly
+
+You can create your inequalities directly, then solve the system as a list:
+
+```py
+>>> from sympy import symbols, solve, pi, Integer
+>>> x = symbols('x')
+>>> one_third = Integer(1)/Integer(3)
+>>> solve([x >= one_third, x**2 <= pi], x)
+(1/3 <= x) & (x <= sqrt(pi))
+```
+
+### Put your inequalities into the Relational class
+
+You can create each inequality using the {class}`Relational class
+<sympy.core.relational.Relational>` by specifying the left-hand side, the
+right-hand side, and then a relational operator such strict greater than (`gt`
+or `>`), or less than or equal to (`le` or `<=`):
+
+```py
+>>> from sympy.core import Rel
+>>> from sympy import symbols, Integer, pi, solve
+>>> x = symbols('x')
+>>> inequality1 = Rel(x, Integer(1)/Integer(3), 'gt')
+>>> inequality1
+x > 1/3
+>>> inequality2 = Rel(x**2, pi, '<=')
+>>> inequality2
+x**2 <= pi
+>>> solve([inequality1, inequality2], x)
+(1/3 < x) & (x <= sqrt(pi))
+```
+
+## Use the solution result
+
+A common way to use the solution result is to extract the bounds for the symbol
+(variable). For example, for a solution of $0 < x < \sqrt{pi}$, you might want
+to extract $0$ and $\sqrt{pi}$.
+
+### Extract relational atoms
```

Comment:
I think we should just omit this section. No one should really use Relational directly like this.